client/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,32 @@
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardAction from '@material-ui/core/CardActions';
```

Comment:
You can merge these imports together to avoid clustering your imports unnecessarily. 
`import { Avatar, Button, ...} from '@material-ui/core';`

Reply:
Done!